book.rb

```diff
@@ -0,0 +1,15 @@
+require_relative 'rentable'
+
+class Book
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(date, person, book)
+    Rental.new(date, person, book)
+  end
```

Comment:
- [ ] [OPTIONAL] Kindly update this method to be like this:
```ruby
  def add_rental(person, date)
    Rental.new(date, person, self)
  end
```
You don't need to pass `book` as a parameter as you already have access to it on `self`.

Reply:
done